scripts/fast-reboot

```diff
@@ -48,7 +48,12 @@ then
         mkdir -p "$FS_MOUNTPOINT"
         mount -t squashfs "$NEXT_IMAGE_FS_PATH" "$FS_MOUNTPOINT"
         /usr/bin/mlnx-fw-upgrade.sh "$FS_MOUNTPOINT/etc/mlnx/fw-SPC.mfa"
+        EXIT_CODE=$?
         umount "$FS_MOUNTPOINT"
+        if [[ $EXIT_CODE != 0 ]]; then
+            echo "Failed to burn FW"
```

Comment:
>echo [](start = 12, length = 4)

Should we check or restore firmware if the upgrading is almost done but failed?
If mlnx-fw-upgrade.sh is super robust and responsible to handle every failure and recovery, I am ok.